src/sentry/static/sentry/app/components/group/suggestedOwners.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+import Avatar from '../avatar';
+import TooltipMixin from '../../mixins/tooltip';
+import ApiMixin from '../../mixins/apiMixin';
+import GroupState from '../../mixins/groupState';
+
+import {t} from '../../locale';
+
+const SuggestedOwners = React.createClass({
+  propTypes: {
+    event: React.PropTypes.object,
+  },
+
+  mixins: [
+    ApiMixin,
+    GroupState,
+    TooltipMixin({
+      selector: '.tip',
+      container: 'body'
+    })
+  ],
+
+  getInitialState(){
+      return {owners: undefined};
+  },
+
+  componentDidMount(){
+    this.fetchData(this.props.event);
+  },
+
+  componentWillReceiveProps(nextProps){
+    if(this.props.event != nextProps.event){
+      this.fetchData(nextProps.event);
```

Comment:
You can't do a reference equality here – since it's common in React to change the reference using `{...event}` or `Object.assign`.

Compare using the ID or some other unique property.